and-bible/app/src/main/java/net/bible/android/view/activity/page/MainBibleActivity.java

```diff
@@ -113,6 +116,42 @@ public void onCreate(Bundle savedInstanceState) {
 
 		// force the screen to be populated
 		PassageChangeMediator.getInstance().forcePageUpdate();
+		refreshScreenKeepOn();
+		checkSdcardReadPermission();
```

Comment:
Yes, only for that purpose. I agree with you, I was thinking about this too. But then when @ildar mentioned about it I wanted just implement something as this had been left hanging unimplemented for quite some time.

So what do you think, should I add new boolean setting in settings screen
"Read modules from /sdcard/jsword"?

- [ ] change implementation

Reply:
I think it might be better.  We could ask in the discussion group.  Then changing the setting could immediately invoke the request for permission to write to the SD Card and the request could be removed from startup.

Reply:
Created new setting for this and changed implementation as discussed.